pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -87,7 +87,9 @@ func (r *ReconcileClusterLogging) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, nil
 	}
 
-	err = k8shandler.Reconcile(instance, r.client)
+	if err = k8shandler.Reconcile(instance, r.client); err != nil {
+		logrus.Error(err, "Error reconciling clusterlogging instance")
```

Comment:
This error should be returned or recorded in status, not dropped.

Reply:
Agreed. I think that will be addressed by @sasagarw work to fix up our status.  We may want to pull that back into 4.6 as a stability item.  Do we need to block this backport here?

Reply:
no, if it's not making things worse

Reply:
/lgtm